otel/opentelemetry-instrumentation-langchain/opentelemetry/instrumentation/langchain/otel_lib/tracer.py

```diff
@@ -421,12 +421,12 @@ def _persist_run(self, run: Run) -> None:
         try:
             self._convert_run_to_spans(run.dict(), modelmetrics)
             for modelmetric in modelmetrics:
-                token_counter = self.meter.create_up_down_counter(f"{modelmetric.llm}_{modelmetric.model}_token")
-                call_counter = self.meter.create_up_down_counter(f"{modelmetric.llm}_{modelmetric.model}_calls")
-                duration_counter = self.meter.create_up_down_counter(f"{modelmetric.llm}_{modelmetric.model}_duration")
-                token_counter.add(modelmetric.token)
-                call_counter.add(modelmetric.calls)
-                duration_counter.add(modelmetric.duration)
+                token_counter = self.meter.create_up_down_counter(f"llm.request.token")
+                call_counter = self.meter.create_up_down_counter(f"llm.request.count")
+                duration_counter = self.meter.create_up_down_counter(f"llm.response.duration.avg")
+                token_counter.add(modelmetric.token, {"model_id": modelmetric.model, "user_id": "user1"})
```

Comment:
for the `"user_id": "user1"`, I don't think we can capture this user_id data in our instrumentation.